faithcomesbyhearing/dbp
# FCBH-759 Add example API responses to all endpoints

# Description
- Updated the example response object of the endpoint to match the actual result

## Issue Link
Original Story: [FCBH-759](https://fullstacklabs.atlassian.net/browse/FCBH-759) 
Review Story: [FCBH-1396](https://fullstacklabs.atlassian.net/browse/FCBH-1396) 

## How Do I QA This
- On your local environment run `https://dbp.test/open-api-v4.json` and `https://dbp.test/open-api-v2.json` and verify the example responses are updated

## Checklist
